eos/form-factors/parametric-bfw2010-impl.hh

```diff
@@ -451,47 +451,129 @@ namespace eos
 
     template<typename Process_>
     double
-    BFW2010FormFactors<Process_, PToV>::saturation_1m_v() const
+    BFW2010FormFactors<Process_, PToV>::saturation_0m_a() const
     {
-        return std::inner_product(_a_V.begin(), _a_V.end(), _a_V.begin(), 0.0);
+        return std::inner_product(_a_A0.begin(), _a_A0.end(), _a_A0.begin(), 0.0);
     }
 
     template<typename Process_>
     double
-    BFW2010FormFactors<Process_, PToV>::saturation_0m_a() const
+    BFW2010FormFactors<Process_, PToV>::saturation_1m_v_0() const
     {
-        return std::inner_product(_a_A0.begin(), _a_A0.end(), _a_A0.begin(), 0.0);
+        return 0.0;
     }
 
     template<typename Process_>
     double
-    BFW2010FormFactors<Process_, PToV>::saturation_1p_a() const
+    BFW2010FormFactors<Process_, PToV>::saturation_1m_v_perp() const
+    {
+        return 3.0 * std::inner_product(_a_V.begin(), _a_V.end(), _a_V.begin(), 0.0);
```

Comment:
The factor of three is convention here, is a bit weird and requires a comment I think.